lib/AST/ASTPrinter.cpp

```diff
@@ -2047,6 +2053,27 @@ void PrintAST::visitExtensionDecl(ExtensionDecl *decl) {
     printExtension(decl);
 }
 
+void PrintAST::printPatternBindingEntry(const PatternBindingEntry &entry) {
+  printPattern(entry.getPattern());
+
+  // We also try to print type for named patterns, e.g. var Field = 10;
+  // and tuple patterns, e.g. var (T1, T2) = (10, 10)
+  if (isa<NamedPattern>(entry.getPattern()) ||
+      isa<TuplePattern>(entry.getPattern())) {
+    printPatternType(entry.getPattern());
+  }
+
+  if (Options.VarInitializers) {
+    // FIXME: Implement once we can pretty-print expressions.
+  }
```

Comment:
Hm, this `if` should probably be guarding the next section…

Reply:
Yeah, this was left over from the original implementation. I'll just go ahead and change it. Should we even have the option to turn it off, now that var initializers are part of the public interface anyway?

Reply:
I…guess not? Although I suppose we could have an option that chooses between "all of them" and "only the ones that are public".